site/content/en/docs/manual/advanced/delete_frame.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Frame deleting'
+linkTitle: 'Delete frame from task'
+weight: 18
+description: 'This section explains how delete and restore frame from task.'
+---
+
+# Delete frame
+
+You can delete current frame from task. This frame will not be presented either in the UI or in the exported annotation.
+Thus, it is possible to mark corrupted frames that are not subject to annotation.
+
+1. Go to the Job annotation view and than click on the `Delete frame` button.
+
+   ![](/images/image245.jpg)
+
+1. After that you will be asked to confirm frame deleting.
+   **Important note:**  all annotaiton from that frame will be deleted, unsaved annotations
+   will be saved and frame will be unvisable in the annotation view (Until you make it visible in the settings).
+   If there is some overlap in the task and the deleted frame falls within this interval,
+   then this will cause this frame to become unavailable in another job as well.
+1. If you annotate interpolation task, you may need to make some correction in the annotation. Common ones are:
```

Comment:
When you delete a frame in a job with tracks, you may need adjust some tracks if the deleted frame has key frames for these tracks. You will need to adjust these tracks manually.

Reply:
Changed. But it also can relate with any deleted frames not only ones with keyframes.